### DISCUSSION TITLE
cant install motion 4.6 on raspberry arm v6

### DISCUSSION THREAD

**Discussion by TheDarkRiddle** [Motion-Project/motion#1766] [Q&A]
Hi there, i have motion 4.3.2 (4.5.1 is running also fine)  running fine on a older pi model B with older cpu:
LSCPU:
Architecture:        armv6l
Byte Order:          Little Endian
CPU(s):              1
On-line CPU(s) list: 0
Thread(s) per core:  1
Core(s) per socket:  1
Socket(s):           1
Vendor ID:           ARM
Model:               7
Model name:          ARM1176
Stepping:            r0p7
CPU max MHz:         700.0000
CPU min MHz:         700.0000
BogoMIPS:            697.95
Flags:               half thumb fastmult vfp edsp java tls

i like to update to version 4.6 and try this via "wget" and "gdebi"
For my older cpu i try all releases ending with "....armhf.deb" from the projeectpage.
either i see "This package is uninstallable" or i see "Segmentation fault" if i try running motion after instaltaion.
a logflie is not written!
what i need to do that i can run the new motion mo my older pi? or shall i stay with the older 4.3.2?

**Comment by tosiara** [comment]
Did you try  `pi_bookworm_motion_4.6.0-1_armhf.deb` ?
